web/js/FlexyConnector.js

```diff
@@ -26,7 +26,8 @@ function saveJSON()
    //Fetch the AppConfig
    var CycleTimeMs = document.getElementById("CycleTimeMs").value;
    var PostDuplicateTagValues = document.getElementById("PostDuplicateTagValues").value;
-   jsonObject.AppConfig = { CycleTimeMs: CycleTimeMs, PostDuplicateTagValues: PostDuplicateTagValues };
+   var OMF = document.getElementById("OMF").value;
+   jsonObject.AppConfig = { CycleTimeMs: CycleTimeMs, PostDuplicateTagValues: PostDuplicateTagValues, OMF: OMF };

```

Comment:
For the javascript did you test it? It looks like you are missing some bits. at first glance you are missing functionality in saveJSON()

Reply:
it works when i tried it. my only complaint is the boolean values are in quotes but it looks like it would take me a while to change that and it doesn't really matter.